miqa/core/models/project.py

```diff
@@ -39,6 +39,13 @@ class Project(TimeStampedModel, models.Model):
     archived = models.BooleanField(default=False)
     import_path = models.CharField(max_length=500, blank=True)
     export_path = models.CharField(max_length=500, blank=True)
+    neurology_orientation = models.BooleanField(
```

Comment:
Change to `CharField` with `TextChoices` that directly indicate the anatomical orientation.